gwt-material/src/main/java/gwt/material/design/client/base/mixin/EnabledMixin.java

```diff
@@ -69,21 +67,16 @@ private void applyEnabledProperty(boolean enabled, UIObject obj) {
             obj.removeStyleName("disabled");
             obj.getElement().removeAttribute(DISABLED);
 
-            if(uiObject instanceof HasWaves) {
-                if(((HasWaves) uiObject).getWaves() == null) {
-                    uiObject.addStyleName(Waves.WAVES_STYLE);
-                    Waves.detectAndApply();
-                }
-            }
         } else {
             obj.addStyleName("disabled");
             obj.getElement().setAttribute(DISABLED, "");
 
-            if(uiObject instanceof HasWaves) {
-                if(((HasWaves) uiObject).getWaves() != null) {
-                    uiObject.removeStyleName(((HasWaves) uiObject).getWaves().getCssName());
+        }
+        if(obj instanceof HasWidgets) {
+            for(Widget widget: (HasWidgets) obj) {
```

Comment:
This will break the Tab component , specially the design. Because Link has default disabled styles that will override because you loop the entire Widget then set it to disable.